src/test/java/integration/DiscountControllerIT.java

```diff
@@ -141,22 +140,21 @@ public void updateDiscount() throws Exception {
         when().
                 put("/discounts/{uuid}", discountUuid).
         then().
-                statusCode(SC_OK);
-        long timeFinished = currentTimeMillis();
+                statusCode(SC_OK).
+                body("fromDate", equalTo(inputJson.get("fromDate"))).
+                body("toDate", equalTo(inputJson.get("toDate")));
+        LocalDateTime timeFinished = now();
 
         // check updated data
         JSONParser parser = new JSONParser();
         JSONObject expectedJson = (JSONObject) parser.parse(inputJson.toJSONString());
 
         Discount updatedDiscount = discountService.getEntityByUuid(discountUuid);
-        long timeModified = updatedDiscount.getLastModified().getTime();
+        LocalDateTime timeModified = updatedDiscount.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).